robottelo/libvirt_discovery.py

```diff
@@ -56,7 +56,13 @@ def __init__(
         else:
             self.image_dir = image_dir
         if mac is None:
-            self.mac = gen_mac(multicast=False, locally=True)
+            # fe:* MAC range is considered reserved in libvirt
+            while True:
+                mac = gen_mac(multicast=False, locally=True)
+                if not mac.startswith(u'fe'):
```

Comment:
is that `if not` correct? in your comment seems to be only `if`.

Also, what happens if a mac starting with `fe` is never generated?